src/rgw/rgw_sync.h

```diff
@@ -420,18 +426,13 @@ class RGWMetaSyncSingleEntryCR : public RGWCoroutine {
 
   bool error_injection;
 
+  RGWSyncTraceNodeRef tn;
+
 public:
   RGWMetaSyncSingleEntryCR(RGWMetaSyncEnv *_sync_env,
 		           const string& _raw_key, const string& _entry_marker,
                            const RGWMDLogStatus& _op_status,
```

Comment:
while you're at it can you fix the alignment here, (not a part of this changeset)